proto/store/workspace_setting.proto

```diff
@@ -75,6 +75,7 @@ message StorageS3Config {
   string endpoint = 3;
   string region = 4;
   string bucket = 5;
+  string prefix_url = 6;
```

Comment:
> > Presigned URLs can only be used with the .r2.cloudflarestorage.com S3 API domain and cannot be used with custom domains. Instead, you can use the [general purpose HMAC validation feature of the WAF](https://developers.cloudflare.com/ruleset-engine/rules-language/functions/#hmac-validation), which requires a Pro plan or above.预签名 URL 只能用于 .r2.cloudflarestorage.com S3 API 域，不能用于自定义域。相反，您可以使用 WAF 的通用 HMAC 验证功能，该功能需要 Pro 计划或更高版本。
> 
> Maybe presigned urls cannot fit the custom domain in cloudflare r2. Did you test this locally?也许预签名的 url 无法适应 cloudflare r2 中的自定义域。你在本地测试过吗？
> 
> https://developers.cloudflare.com/r2/api/s3/presigned-urls/#limitations

Yes, for R2, presigned URLs cannot be used with custom domains. Therefore, when setting the prefixURL, the file path is composed of prefixURL/Filepath without using the presigned feature. 
I have tested it on both local and VPS deployed server environments, and it can display images correctly in both cases.